fa_crosshair/init.sqf

```diff
@@ -33,9 +34,17 @@ dc_fac_fnc_crosshairInit = {
 		if (isNil {profileNamespace getVariable "dc_fac_var_crosshairScale"}) then {
 			2 call dc_fac_fnc_setCrosshairScale;
 		};
-		
+		if (isNil {profileNamespace getVariable "dc_fac_var_crosshairEnable"}) then {
+			profileNamespace setVariable ['dc_fac_var_crosshairEnable',true];
+		};
+
+
 		_briefingText = "";
 
+		_briefingText = _briefingText + format ["
+			<br/><execute expression=""profileNamespace setVariable ['dc_fac_var_crosshairEnable',true]"">Turn On Crosshair</execute>

```

Comment:
This can probably be condensed. Something like Enable Crosshair: ON OFF, where ON/OFF are links. Or as a toggle, but that might be bad UX since there isn't really a way to indicate the current option.